scrapers/PaperStreetMedia.yml

```
@@ -37,8 +38,32 @@ xPathScrapers:
         Name:
           selector: //link[@rel="canonical"]/@href
           postProcess:
-              - replace: 
-                  - regex: (.+www\.)(\w+)(.+)
-                    with: $2
+            - replace: 
+                - regex: (.+www\.)(\w+)(.+)
+                  with: $2
+            - map:
+                badmilfs: BadMilfs
+                bbcparadise: "BBC Paradise"
+                bffs: Bffs
```

The `f`s should be capitalized here

```suggestion
                bffs: BFFs
```